README.md

```diff
@@ -58,47 +61,144 @@ Instructions on how to install and deploy JupyterHub SSH & SFTP services.
 
 1. Start the JupyterHubSSH app from the directory where the config file
    `jupyterhub_ssh_config.py` is located:
-   `bash $ python -m jupyterhub_ssh `
 
-### Kubernetes based deployment
-
-If your JupyterHub was deployed using Kubernetes, you can use the Helm charts available in this repo to deploy JupyterHub SSH & SFTP
-directly into your Kubernetes cluster.
+   ```bash
+   python -m jupyterhub_ssh
+   ```
 
-- Let helm the command line tool know about a Helm chart repository that we decide to name jupyterhub.
-  ```bash
-  $ helm repo add jupyterhub-ssh https://yuvipanda.github.io/jupyterhub-ssh/
-  $ helm repo update
-  ```
-- Simplified example on how to install a Helm chart from a Helm chart repository named jupyterhub-ssh. See the Helm chart's documentation
-  for additional details required.
-  ```bash
-  $ helm install jupyterhub-ssh/jupyterhub-ssh --version <helm chart version> --set hubUrl=https://jupyter.example.org --set-file hostKey=<path to a private SSH key>
-  ```
+### Kubernetes based deployment (jupyterhub ssh and/or sftp)
+
+If your JupyterHub has been deployed to Kubernetes, you can use the Helm chart
+available in this repo to deploy JupyterHub SSH and/or JupyterHub SFTP directly
+into your Kubernetes cluster.
+
+```bash
+helm install <helm-release-name> \
+   --repo https://yuvipanda.github.io/jupyterhub-ssh/ jupyterhub-ssh \
+   --version <helm chart version> \
+   --set hubUrl=https://jupyter.example.org \
+   --set ssh.enabled=true \
+   --set sftp.enabled=false
+```
+
+If you install JupyterHub SFTP, then it needs access to the home folders. These
+home folders are assumed to be exposed via a k8s PVC resource that you should
+name via the `sftp.pvc.name` configuration.
+
+If your JupyterHub has been deployed using [the official JupyterHub Helm
+chart](https://z2jh.jupyter.org) version 1.1.0 or later, and you have
+_configured the official JupyterHub Helm chart_ with `proxy.https.enabled=true`
+and `proxy.https.type=letsencrypt`, then you can add the following to to acquire
+access to the jupyterhub-ssh and jupyterhub-sftp services via that setup.
+
+```yaml
+# Configuration for the official JupyterHub Helm chart to accept traffic via
+proxy:
+   https:
+      enabled: true
+      type: letsencrypt
+      letsencryptEmail: <my-email-here>
+
+   service:
+     # jupyterhub-ssh/sftp integration part 1/3:
+     #
+     # We must accept traffic to the k8s Service (proxy-public) receiving traffic
+     # from the internet. Port 22 is typically used for both SSH and SFTP, but we
+     # can't use the same port for both so we use 23.
+     #
+     extraPorts:
+       - name: ssh
+         port: 22
+         targetPort: ssh
+       - name: sftp
+         port: 23
```

Comment:
23 is the port allocated to telnet (https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers), so let's use something else? I prefer 2222, since sftp is normally in 22.